common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2968,8 +2973,10 @@ def _copy_subdag(self, user_id, destination_version, block_key, source_blocks, d
                 if getattr(destination_block.edit_info, key) is None:
                     setattr(destination_block.edit_info, key, val)
 
-        # introduce new edit info field for tracing where copied/published blocks came
-        destination_block.edit_info.source_version = new_block.edit_info.update_version
+        #If the block we are copying from was itself a copy, then just reference the original source,
+        # rather than the copy.
+        destination_block.edit_info.source_version = (new_block.edit_info.source_version or
+                                                      new_block.edit_info.update_version)
```

Comment:
@cpennington Does a `source_version` of `None` always indicates a copied block? If so, could that fact be added to the docstring at the top of split.py? I'd like to tighten up that help - there's confusion around the numerous attributes in edit_info.